reference/carts.v3.yml

```diff
@@ -1679,35 +1692,32 @@ components:
                                 expiresDate:
                                   type: integer
                                   example: 0
-                                  description: Returns 0 if a expiration date is not set
+                                  description: Returns 0 if no expiration date has been set.
                                 totalDiscount:
                                   type: number
                                   example: 4.19
-                                  description: Total discount amount off cart
+                                  description: Total amount discounted off the entired cart
                           required:
                             - coupon_code
                   discount_amount:
                     type: number
-                    description: The total value of all discounts applied to this item. This includes coupons and cart level discounts
+                    description: The total value of all discounts applied to this item. This includes coupons and cart level discounts.
                     example: 4
                   coupon_amount:
                     type: number
                     description: The total value of all coupons applied to this item.
-                  original_price:
-                    type: number
-                    description: An item’s original price is the same as the product default price in the admin panel.
                   list_price:
                     type: number
-                    description: 'The net item price before discounts and coupons. BigCommerce derives an item’s list price from the product default price or, if applicable, the sale price configured in the admin panel.'
+                    description: 'List price of the item, as quoted by the manufacturer/distributor. Optional price override.'
```

Comment:
@bc-traciporter this is less clear.

Reply:
@slsriehl  These changes were made in [PR 896](https://github.com/bigcommerce/api-specs/pull/896). @bc-shawnwang and @bc-mackxu were involved with these changes.